react-toggle-switch-component/src/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+class ToggleSwitch extends React.Component {
```

Comment:
Move the component class definition into a file named `toggle-switch.jsx`

Reply:
![image](https://user-images.githubusercontent.com/71291742/137420056-eff2fbce-04d9-4df7-ad67-93352e355eca.png)